numpy/__init__.cython-30.pxd

```diff
@@ -0,0 +1,1100 @@
+# NumPy static imports for Cython
+#
+# If any of the PyArray_* functions are called, import_array must be
+# called first.  This is done automatically by Cython 3.0+ if a call
+# is not detected inside of the module.
+#
+# Author: Dag Sverre Seljebotn
```

Comment:
Can you adjust the header in the other file to explain that it only used on old versions of cython?

Reply:
Done.